internal/pkg/service/buffer/sink/tablesink/storage/repository/file_repo.go

```diff
@@ -128,9 +161,9 @@ func (r *FileRepository) get(k storage.FileKey) op.ForType[*op.KeyValueT[storage
 // The entity is stored in 2 copies, under "All" prefix and "InLevel" prefix.
 // - "All" prefix is used for classic CRUD operations.
 // - "InLevel" prefix is used for effective watching of the storage level.
-func (r *FileRepository) put(v storage.File, create bool) *op.TxnOp {
-	level := v.State.Level()
-	etcdKey := r.schema.AllLevels().ByKey(v.FileKey)
+func (r *FileRepository) put(oldValue, newValue storage.File, create bool) *op.TxnOp {
```

Comment:
Ok, here is update: https://github.com/keboola/keboola-as-code/pull/1482/commits/8720c5e135cebc8a017bb315600f703142f49418

As I wrote last time, the values are written under 2 keys to optimize the Watch operation.

That's why I preferred to have one `put` method where this common logic was. I don't see a problem with the bool flag in a private method.

But I modified it, let's see if it will be easier for understanding and maitanance. 

Now it is much easier to modify create and forget to modify the update method.